lib/services/notifications_service.rb

```diff
@@ -42,6 +42,21 @@ def create(client_urn, notification, params={auth_token: nil})
       return get_api_response([body], nil, response.code)
     end
 
+    def update(user_id, notification, params={auth_token: nil})
+      auth_token = params[:auth_token] || nil
+      response = RestClient.put(
+        "#{@host}/users/#{user_id}/notifications/#{notification.id}",
+        notification,
```

Comment:
what i had coded on Hub side was that the body of the PUT POST is expected to be: user_notification: { read_at: read_at} (read_at being the only updatable aspect).  if we want future room for notification updating, then the body would be: notification: { user_notification: { read_at: read_at} }